plugins/examples/change-logo/src/index.tsx

```diff
@@ -20,18 +20,19 @@ import Settings, { store } from './settings';
  * it's easier to make it look good with light and dark themes.
  */
 function SimpleLogo(props: AppLogoProps) {
-  const { logoType, className } = props;
+  const { logoType, className, sx } = props;
```

Comment:
I guess there was some sort of error during rebasing and this commit snuck into the PR somehow.